goon/browserassets/js/browserOutput.js

```diff
@@ -92,15 +92,24 @@ if (typeof String.prototype.trim !== 'function') {
 		return this.replace(/^\s+|\s+$/g, '');
 	};
 }
+// Polyfill for endsWith()
+if (typeof String.prototype.endsWith !== 'function') {
+	String.prototype.endsWith = function(search, this_len) {
+		if (this_len === undefined || this_len > this.length) {
+			this_len = this.length;
+		}
+		return this.substring(this_len - search.length, this_len) === search;
+	};
+}
 
 //Polyfill for string.prototype.includes. Why the fuck. Just why the fuck.
 if (!String.prototype.includes) {
 	String.prototype.includes = function(search, start) {
 	  'use strict';
-  
+
```

Comment:
Seems like some whitespace was added or removed due to an extension in VSCode, apologies for the noise.

Reply:
It's perfectly normal, everyone's VSC does it. Mine does anyway.